### PR TITLE
bump ginkgo to v2.0.0

### PR DESCRIPTION
**Pull request by Hexilee**
### What problem does this PR solve?

In #2668, I would like to add some e2e tests that run with a serial decorator, so we need ginkgo v2.

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [x] Unit test
- [x] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: e2e-test/e2e/chaos/stresschaos/memory.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"net/http"
 	"time"
 
-	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/v2"
```
Comment (YangKeao) [inline e2e-test/e2e/chaos/stresschaos/memory.go:24]: Why don't we globally import this package?

Comment (Hexilee) [reply e2e-test/e2e/chaos/stresschaos/memory.go:24]: My linter suggested it to me.

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Comment by Hexilee** [comment]
It seems the ginkgo v2 is too hard to be upgraded, we will do it in later days.

**Pull request closed without merging**